### PR TITLE
docs: miscellaneous README changes

### PR DESCRIPTION
**Pull request by nfischer**
- Add @freitagbr to the bottom of the README
 - Remove docs about shjs (it's not well supported)
 - Remove docs about coffeescript (we don't do anything special for
   coffeescript anyway)
 - Modify the section about shelljs/global, since we strongly recommend
   avoiding this now
 - Rewrite the code example to use `require('shelljs')` instead of
   shelljs/global
 - Mention ESLint next to JSHint
 - Reformat sections with long lines to be 80 columns

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,34 @@
-# ShellJS - Unix shell commands for Node.js
+# ShellJS - Unix shell commands for NodeJS
```
Comment (freitagbr) [inline README.md]: We should stick with "Node.js", as that is the official name.

File: README.md [hunk 2]
```diff
@@ -1,33 +1,34 @@
...
-Node.js API. You can use it to eliminate your shell script's dependency on Unix while still keeping
-its familiar and powerful commands. You can also install it globally so you can run it from outside
-Node projects - say goodbye to those gnarly Bash scripts!
+ShellJS is a portable **(Windows/Linux/OS X)** implementation of Unix shell
+commands on top of the Node.js API. You can use it to eliminate your shell
+script's dependency on Unix while still keeping its familiar and powerful
+commands. You can also install it globally so you can run it from outside Node
+projects - say goodbye to those gnarly Bash scripts!
 
 ShellJS is proudly tested on every node release since `v0.11`!
```
Comment (freitagbr) [inline README.md]: We're only testing on `v4` and up now right? Maybe we should change this to `v4`.

File: README.md [hunk 3]
```diff
@@ -812,6 +777,6 @@ Reset shell.config to the defaults:
 
 ## Team
 
-| [![Nate Fischer](https://avatars.githubusercontent.com/u/5801521?s=130)](https://github.com/nfischer) | [![Ari Porad](https://avatars1.githubusercontent.com/u/1817508?v=3&s=130)](http://github.com/ariporad) |
+| [![Nate Fischer](https://avatars.githubusercontent.com/u/5801521?s=130)](https://github.com/nfischer) | [![Brandon Freitag](https://avatars1.githubusercontent.com/u/5988055?v=3&s=130)](http://github.com/freitagbr) |
```
Comment (freitagbr) [inline README.md]: :+1:

File: README.md (hunk 2 above)
Comment (nfischer) [inline README.md]: That hasn't been merged to master branch yet, only dev. And I think that README change already exists in the dev branch.

We could just as well base this PR off dev, instead. No hurry.

File: README.md (hunk 1 above)
Comment (nfischer) [inline README.md]: Ack, will fix

File: README.md (hunk 2 above)
Comment (freitagbr) [inline README.md]: Ah, right. Let's leave this as v0.11 for now.

**Comment by nfischer** [comment]
@freitagbr PTAL. I changed `NodeJS` ➡️ `Node.js` in two places where it appeared.

**Comment by freitagbr** [comment]
One of the "NodeJS" occurrences is in the `src/exec.js` docs, so it needs to be fixed there too.

**Comment by nfischer** [comment]
@freitagbr PTAL. Thanks for catching this

**Comment by freitagbr** [comment]
LGTM :+1:

**Pull request merged**